client/tests/jest/helpers.js

```diff
@@ -146,7 +146,13 @@ export const showAll = (vm) => {
 
 // waits n milliseconds and then promise resolves
 // usage: await wait(500);
-export const wait = (n) => timer(n).pipe(take(1)).toPromise();
+export const wait = (n) => {
```

Comment:
Thanks for this -- always feel free to cut out any of the rxjs-isms you find; I'd like to get rid of it eventually.